INFO     template_model.conftest:conftest.py:248 [2025-10-15 18:36:42] 用例 test_community_vps/test_delete.py::TestDeleteUser::test_delete_userlist 开始登录，环境: dev
DEBUG    urllib3.connectionpool:connectionpool.py:243 Starting new HTTP connection (1): 39.99.136.49:80
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://39.99.136.49:80 "POST /api/sys/auth/login HTTP/11" 200 None
INFO     template_model.conftest:conftest.py:275 测试环境登录成功
INFO     template_model.commons.variable_manager:variable_manager.py:111 [2025-10-15 18:36:36] 运行时变量已保存到: D:\pycharm_test\template_model\VAR\runtime_vars_community.json
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://39.99.136.49:80 "DELETE /api/mascontrol/user HTTP/11" 200 None
INFO     template_model.commons.api_vpsbase:api_vpsbase.py:239 [2025-10-15 18:36:42] DELETE请求: /mascontrol/user | 数据: [7815]
INFO     root:test_delete.py:202 第1个账号（ID: 7815）删除接口调用成功
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://39.99.136.49:80 "DELETE /api/mascontrol/user HTTP/11" 200 None
INFO     template_model.commons.api_vpsbase:api_vpsbase.py:239 [2025-10-15 18:36:43] DELETE请求: /mascontrol/user | 数据: [7816]
INFO     root:test_delete.py:202 第2个账号（ID: 7816）删除接口调用成功
INFO     template_model.conftest:conftest.py:343 [2025-10-15 18:36:44] 用例 test_community_vps/test_delete.py::TestDeleteUser::test_delete_userlist 执行完毕